README.md

```diff
@@ -25,6 +25,14 @@ The build validation scripts are available as pre-built, executable Bash scripts
 * how to use the [build validation scripts for Gradle](Gradle.md)
 * how to use the [build validation scripts for Maven](Maven.md)
 
+## Gradle Enterprise version compatibility
+This table details the version compatibility of the Build Validation Scripts with the Gradle Enterprise. Some scripts fetch data from build scans that were published as part of running an experiment. Fetching build scan data only works for compatible versions of Gradle Enterprise.
+
+| Build Validation Scripts versions | Gradle Enterprise versions |
+|-----------------------------------|----------------------------|
+| `1.0 - 1.0.2`                     | `2021.2+`                  |
+| `2.0`                             | `2022.2.4+`                |
```

Comment:
Why is such a recent version of GE required?

Reply:
This was actually wrong. I had forgotten that we haven't yet merged PR #98 (which takes advantage of the "non-cacheable reason" that was added to the Gradle Enterprise API in 2022.2.4).

I corrected this to 2022.1, which is the version we added support for returning the remote build cache URL to the API.